src/datas/datas.js

```diff
@@ -0,0 +1,6 @@
+const datas = {
```

Comment:
```suggestion
const MESSAGES = Object.freeze({
```
런타임 중에 변하지 않는 상수 객체이니깐 컨벤션을 스네이크 케이스 사용해도 좋을 것 같아요👍

Reply:
아하 감사합니다! 
꼼꼼한 리뷰 정말 감사드립니다 
배우고 가는 점이 정말 많네요 .. !
다음 주차에 꼭 반영하도록 하겠습니다 👍